src/dstack/_internal/core/backends/datacrunch/compute.py

```diff
@@ -36,9 +36,7 @@ def get_offers(
     def _get_offers_with_availability(
         self, offers: List[InstanceOffer]
     ) -> List[InstanceOfferWithAvailability]:
-        raw_availabilities: List[
-            Dict
-        ] = self.api_client.client.instances.get_availabilities()  # type: ignore
+        raw_availabilities: List[Dict] = self.api_client.client.instances.get_availabilities()  # type: ignore
```

Comment:
It seems `# type: ignore` should be removed and get_availabilities() declaration be fixed.

Reply:
Thanks, fixed!